patrick/movies/src/containers/MovieDetails.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import { getMovieDetailsByID } from '../actions';
+
+class MovieDetails extends Component {
+  constructor(props) {
```

Comment:
No needs to pass `props` to `constructor` here since `props` aren't being used inside the constructor.